lib/tableau/extensions/post_extension.ex

```diff
@@ -129,6 +132,14 @@ defmodule Tableau.PostExtension do
           Enum.reject(posts, fn {post, _} -> DateTime.after?(post.date, DateTime.utc_now()) end)
         end
       end)
+      |> then(fn posts ->
+        if config.drafts do
+          Enum.reject(posts, fn {post, _} -> post.file =~ config.drafts end)
```

Comment:
I believe this would incorrectly match on a published post called `_posts/how_to_use_drafts_in_tableau.md`

To see if a file is inside a directory, you typically use `String.starts_with?(post.file, config.drafts)`

Reply:
I should probably write a test against this case?